scripts/data/skills.js

```diff
@@ -131,7 +131,7 @@ var SKILL_DATA = {
 		"desc": "Grants Enrage to a random ally creature, causing it to permanently increase Attack after taking Attack damage.",
 		"icon": "enrage",
 		"name": "Enrage",
-		"type": "earlyActivation"
+		"type": "activation"
```

Comment:
📝 This gets set by an external script that loads the data from Synapse and maintains these files.  I'll need to update that script to mark this as a regular `activation` skill for this change to be permanent.

Reply:
This actually comes from the presence of `<upkeep>1</upkeep>` in the data.  This was also intentionally made an `earlyActivation` skill in [0f7ac1f](https://github.com/TheSench/SIMSpellstone/commit/0f7ac1fe39732866a0f35ad82a4e940bd8032a3a).

Reply:
Nevermind, `skillTypes.jsonc` is overriding this currently.